frontend/src/Editor/Components/Map/Map.jsx

```diff
@@ -107,7 +110,7 @@ export const Map = function Map({
 
 
   return (
-    <div style={{ width, height, display:parsedWidgetVisibility ? '' : 'none' }} onClick={event => {event.stopPropagation(); onComponentClick(id, component)}} className="map-widget">
+    <div disabled={parsedDisableState} style={{ width, height, display:parsedWidgetVisibility ? '' : 'none' }} onClick={event => {event.stopPropagation(); onComponentClick(id, component)}} className="map-widget">
```

Comment:
<div> elements do not have a disabled attribute

Reply:
yes, I know. setting it as a new attribute for the div so that can use it as selectors.
Do you have a better way of this?

Reply:
Yes, we can use `data-*` attributes

more info: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
thanks 😁